description.md

```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easier digestible format of presenting information.
+As such, it is typically used in conjunction with other content types, such as reading, slides or demos.
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
```

Comment:
It is worth mentioning that slides are better than gifs when it comes to illustrating processes or sequential actions. Slides enable students to go at their own pace and also require no additional infrastructure.